packages/riipen-ui-docs/src/pages/components/typography/Color.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import Typography from "riipen-ui/components/Typography";
+
+export default function Color() {
+  return (
+    <div>
+      <Typography color="initial" gutter>
+        Initial
+      </Typography>
```

Comment:
are we moving to one singular color for text? I know we use the _muted_ variant a fair bit currently and there is no real replacement in these new colors

Reply:
That's a good question. Maribel didn't provide colors in her latest guide. I am going to check with her before we merge this.

Reply:
@bolinkd check out https://github.com/riipen/ui/pull/92/commits/3cb51ea5c95d334d56da01af1862d25a0513168d !